toolchain/check/convert.cpp

```diff
@@ -1634,13 +1634,16 @@ auto Convert(Context& context, SemIR::LocId loc_id, SemIR::InstId expr_id,
       if (target.kind == ConversionTarget::RefParam) {
         // Don't diagnose a non-reference scrutinee if it has a user-written
         // `ref` tag, because that's diagnosed in `Convert`.
```

Comment:
We're in `Convert` :)
```suggestion
        // Don't diagnose a non-reference scrutinee if it has a user-written
        // `ref` tag, because that's diagnosed in `CheckRefTag`.
```
(Github stripped the whitespace off the suggested text again, so I predict applying this suggestion will go wrong again. The diff formatting is not consistently working for me either; if it breaks for you too, the change here is replacing `Convert` with `CheckRefTag`.)

Reply:
Good catch, thanks!